src/browser-sandbox/pruneNonCriticalCss.js

```diff
@@ -132,40 +127,53 @@ export default function pruneNonCriticalCss ({
   }
 
   function isCssRuleCritical (rule) {
-    if (rule.type === 'rule') {
+    if (rule.type === 'Rule') {
       // check what, if any selectors are found above fold
-      rule.selectors = rule.selectors.filter(isSelectorCritical)
-      return rule.selectors.length > 0
+      const selectors = rule.prelude.value.split(',')
```

Comment:
That's incorrect approach, since pseudo classes, attribute selector value and comments can contain a comma (e.g. `:not(a, b) /* comme,nt */ [foo="hello, world"]`).
It's better to parse a selector and filter on tree traversal.

Reply:
(moving this comment here where it's better placed):